app/views/govuk_publishing_components/components/_document_list.html.erb

```diff
@@ -17,14 +17,28 @@
 
   brand ||= false
   brand_helper = GovukPublishingComponents::AppHelpers::BrandHelper.new(brand)
+
+  li_data_attributes ||= {}
+
+  disable_ga4 ||= false
+  unless disable_ga4
+    ga4_data ||= {}
+    li_data_attributes[:module] = li_data_attributes[:module] ? li_data_attributes[:module] << " ga4-link-tracker" : "ga4-link-tracker"
+    li_data_attributes[:ga4_track_links_only] = ""
```

Comment:
Just need to check for my own understanding - why is `track-links-only` needed here?

Reply:
@andysellick The `<li>` can contain elements other than links, so using `track-links-only` will cancel the link tracking code at an early stage if the element is not a link or within. If this attribute doesn't exist it will try and run some extra lines of code, so I guess it's a minor optimisation. Though I could remove this, and just put the tracking directly on the two link elements within the list.

Reply:
I think it would make more sense to apply the tracking directly to the links instead of the list items but let's check and see how much effort would be involved first. If it's a straight switch then I'd suggest go for it, but otherwise happy to discuss.

Second potential rabbit hole is the `track-links-only` option and why we have it at all instead of just rewriting the code to only track link clicks when the data attributes are on the parent. But again out of scope of this PR and maybe something for me to look at.

Reply:
Thanks, turns out we can put everything on the `<ul>` so I've done that instead.

I think `track-links-only` came from the UA tracking, I think it was our initial solution to track links rendered within `govspeak` without the tracking firing on non-link elements.

Reply:
Ah cool in that case `track-links-only` works well here, nice work 👍